src/distilabel/pipeline/local.py

```diff
@@ -132,7 +121,7 @@ def run(
         write_buffer = _WriteBuffer(buffer_data_path, self.dag.leaf_steps)
 
         num_processes = len(self.dag)
-        ctx = mp.get_context(_MULTIPROCESSING_CONTEXT)  # type: ignore
+        ctx = mp.get_context()  # type: ignore
```

Comment:
Won't this fail if no `start_method` is set?

Reply:
no, it uses the default one